.gitlab-ci.yml

```diff
@@ -168,7 +168,7 @@ test-on-netgear-rax40:
       - |
         if [ "$DEVICE_UNDER_TEST" != prplmesh ] ; then
             echo "Deploying to $DEVICE_UNDER_TEST"
-            tools/deploy_ipk.sh $DEVICE_UNDER_TEST build/$DEVICE_UNDER_TEST/*.ipk
+            tools/deploy_ipk.sh --certification-mode $DEVICE_UNDER_TEST build/$DEVICE_UNDER_TEST/*.ipk
```

Comment:
Now this commit message makes it clear why we need this flag - I would prefer to make it clear in the PR description and the commit which initially adds this flag.